data/names.txt

```diff
@@ -1029,6 +1060,19 @@ phrase "standalone civilian ship names"
 		"Tiamat"
 		"Artemis"
 		"Annabel Lee"
+		"Old Ironsides"
+		"20 Chanri"
```

Comment:
We sure putting digits in (human) ship names is okay?

Reply:
Why wouldn't it be?

Reply:
Just that no other human ship name follows that pattern, as far as I know. They all use the full word forms, not digit forms.

Reply:
it feels more [robot-like? non-personal?] to me, but a few of them shouldn't be a problem